src/eventTile.ts

```diff
@@ -0,0 +1,87 @@
+import { createDomElement, appendChildren } from "./utils.js";
```

Comment:
Why all the imports leads to `js` files? You should import TS only. Furthermore, it is best practice that bundler (in this TypeScript compiler) would output only 1 file instead of n files (n = total TS files in your project)
 
```suggestion
import { createDomElement, appendChildren } from "./utils";
```